src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexBuilder.cs

```diff
@@ -294,6 +317,12 @@ internal SymbolicRegexNode<TElement> MkSequence(TElement[] seq, bool topLevel)
         /// <returns></returns>
         internal SymbolicRegexNode<TElement> MkNot(SymbolicRegexNode<TElement> node) => SymbolicRegexNode<TElement>.MkNot(this, node);
 
+        internal SymbolicRegexNode<TElement> MkCapture(SymbolicRegexNode<TElement> child, int captureNum) => MkConcat(MkCaptureStart(captureNum), MkConcat(child, MkCaptureEnd(captureNum)));
```

Comment:
NIT: `MkCapture` -> `MakeCaptureNode`. It's fine to keep all of these as is for now for consistency, and I'll later rename most of this to a more descriptive name in a subsequent PR.